flagging_site/config.py

```diff
@@ -61,13 +61,22 @@ def __repr__(self):
 
     @property
     def SQLALCHEMY_DATABASE_URI(self) -> str:
-        user = self.POSTGRES_USER
-        password = self.POSTGRES_PASSWORD
-        host = self.POSTGRES_HOST
-        port = self.POSTGRES_PORT
-        db = self.POSTGRES_DBNAME
-        return f'postgres://{user}:{password}@{host}:{port}/{db}'
-
+        if (self.OFFLINE_MODE == True):
+            user = self.POSTGRES_USER
+            password = self.POSTGRES_PASSWORD
+            host = self.POSTGRES_HOST
+            port = self.POSTGRES_PORT
+            db = self.POSTGRES_DBNAME
+            return f'postgres://{user}:{password}@{host}:{port}/{db}'
+        else:
+            return 'postgres://yuqwhsktykmrqa:34cec8b5de36ee64db9bbc6e6390e1a9'\
+                'ab961008cfcb1640931cd8199fedd971@ec2-34-232-212-164.compute-1'\
```

Comment:
let's discuss at the meeting, but TLDR: this is insecure and you should delete this commit from Github.

```
git checkout dev_postgres
git reset --hard upstream/dev_postgres
git push origin dev_postgres --force
```